apps/ui/src/views/Create.vue

```diff
@@ -76,7 +76,9 @@ const metadataForm: SpaceMetadata = reactive(
     delegations: []
   })
 );
-const selectedNetworkId: Ref<NetworkID> = ref(enabledNetworks[0]);
+const selectedNetworkId: Ref<NetworkID> = ref(
+  enabledNetworks.filter(n => !offchainNetworks.includes(n))[0]
+);
```

Comment:
We have a bit different check here:
https://github.com/snapshot-labs/sx-monorepo/blob/5d7b45b79f308c011afd22ae7e464e10b8deeff2/apps/ui/src/components/FormNetwork.vue#L10-L15

(this is for the list of actually selectable networks in this form)

Those should be in sync so it's better to have shared source of truth for both in case they get out of sync in the future too. Not sure what the best solution would be there. Simplest solution would be to define `enabledReadWriteNetworks` in `networks.ts` and use in both places.